frame/plasm-staking/src/lib.rs

```diff
@@ -158,24 +159,28 @@ decl_storage! {
     trait Store for Module<T: Trait> as PlasmStaking {
         // ----- Staking uses.
         /// Map from all locked "stash" accounts to the controller account.
-        pub Bonded get(fn bonded): map hasher(blake2_256) T::AccountId => Option<T::AccountId>;
+        pub Bonded get(fn bonded): map hasher(blake2_128_concat)
+                                   T::AccountId => Option<T::AccountId>;
         /// Map from all (unlocked) "controller" accounts to the info regarding the staking.
-        pub Ledger get(fn ledger):
-            map hasher(blake2_256) T::AccountId => Option<StakingLedger<T::AccountId, BalanceOf<T>>>;
+        pub Ledger get(fn ledger): map hasher(blake2_128_concat)
+                                   T::AccountId => Option<StakingLedger<T::AccountId, BalanceOf<T>>>;
 
         /// Where the reward payment should be made. Keyed by stash.
-        pub Payee get(fn payee): map hasher(blake2_256) T::AccountId => RewardDestination;
+        pub Payee get(fn payee): map hasher(blake2_128_concat)
+                                 T::AccountId => RewardDestination;
 
         /// The map from nominator stash key to the set of stash keys of all contracts to nominate.
         ///
         /// NOTE: is private so that we can ensure upgraded before all typical accesses.
         /// Direct storage APIs can still bypass this protection.
-        DappsNominations get(fn dapps_nominations): linked_map hasher(blake2_256) T::AccountId => Option<Nominations<T::AccountId>>;
+        DappsNominations get(fn dapps_nominations): map hasher(blake2_128_concat)
```

Comment:
DappsNominators should be `linked_map`.
Because frontend needs to fetch the list of DappsNominators.  The normal `map` can not fetch the key list.

Please let me know if the specifications change.

Reply:
As I can see normal map iterate now, please check it in the code.